rust/flatbuffers/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "flatbuffers"
 version = "0.6.2"
```

Comment:
I think we should bump the version and re-release.

Reply:
> I think we should bump the version and re-release.

What is "re-release" thing?

Reply:
I meant publish

Reply:
I'd like things I commented in [last flatbuffers commit](https://github.com/google/flatbuffers/commit/2eedc769d55cfd538208bff8b3ad5f66bd9239d9#diff-dc7e6bb5dab209b866f71f3bacfc7a05) to be fixed before the release.